modules/search/partials/nav.adoc

```diff
@@ -0,0 +1,33 @@
+  ** xref:7.2@server:search:search.adoc[]
```

Comment:
This is *very deeply nested* compared to many of our other navs - 

I found it a little hard to navigate but my use-case was "Docs Code Reviewer" 
(e.g. I wanted to see where an Asciidoc I was looking at was output in the site, and it was hard to discover it)
so this might be find for actual users?

Reply:
Very deeply nested for good reason, because of where it needs to appear in the nav in Search (under developer guides). Take a look when it's back on Staging and you can see how it looks not just in code :)